api/v1beta1/openstackdataplaneservice_types.go

```diff
@@ -85,6 +85,10 @@ type OpenStackDataPlaneServiceSpec struct {
 	// CACerts - Secret containing the CA certificate chain
 	// +kubebuilder:validation:Optional
 	CACerts string `json:"caCerts,omitempty" yaml:"caCerts,omitempty"`
+
+	// TransNodeset - should the service be deploy across all nodesets
+	// +kubebuilder:validation:optional
+	TransNodeset bool `json:"transNodeset,omitempty" yaml:"transNodeset,omitempty"`
```

Comment:
I'd prefer a different name, that is also more descriptive. Consider:

AllNodeSets
DeployAllNodeSets
DeployAcrossAllNodeSets

Reply:
Should we also make the type `*bool` so that it matches TLSCertsEnabled? Although, we haven't been consistent with that in other Spec structs. Perhaps we should be though.

Reply:
I don't know about the type. I'm personally more on the side of avoiding pointers when it comes to things like `bool`.

Reply:
From what I can tell the ratio between bool and *bool is roughly 50/50. In my opinion the `bool` is better, because you don't have to handle the reference, and since bool is 1 byte it's actually smaller than a pointer would be.

Reply:
Type changed to `*bool` after consultation.